### PR TITLE
README: Rephrase forum into more generic help

### PR DESCRIPTION
**Pull request by bergzand**
### Contribution description

This PR replaces the section called 'Forum' with a more generic section on 'asking for help'. I couldn't find something similar at first glance in our documentation so it is likely helpful for newcomers to explicitly add this.

### Testing procedure

Read the update to the readme and check whether all the links added work.

### Issues/PRs references

None

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -117,12 +117,26 @@ For specific toolchain installation, follow instructions in the
...
 
-## Forum
-Do you have a question, want to discuss a new feature, or just want to present
-your latest project using RIOT? Come over to our [forum] and post to your hearts
-content.
+## Asking for help
 
-[forum]: https://forum.riot-os.org
+You can ask for help on the Forum or on Matrix. Please send bug reports and
+feature requests to our [github issue tracker](https://github.com/RIOT-OS/RIOT/issues)
```
Comment (waehlisch) [inline README.md:123]: GitHub ... .

File: README.md [hunk 2]
```diff
@@ -117,12 +117,26 @@ For specific toolchain installation, follow instructions in the
...
+
+- [forum](https://forum.riot-os.org) is the default place to start asking for
```
Comment (waehlisch) [inline README.md:125]: Forum ... is the default place to ask for help. The Forum provides an archive of prior questions and answers.

File: README.md [hunk 3]
```diff
@@ -117,12 +117,26 @@ For specific toolchain installation, follow instructions in the
...
+help. The search can be used to find older questions and answers.
+- We use [#riot-os:matrix.org](https://matrix.to/#/#riot-os:matrix.org) on the
```
Comment (waehlisch) [inline README.md:127]: For rather synchronous chats, we use the ...

File: README.md (hunk 1 above)
Comment (bergzand) [reply README.md]: Fixed, thanks

File: README.md [hunk 4]
```diff
@@ -117,12 +117,26 @@ For specific toolchain installation, follow instructions in the
...
+[Matrix](https://matrix.org/) chat network for chat-based discussions.
+- [Github Issue tracker](https://github.com/RIOT-OS/RIOT/issues) for issues
```
Comment (waehlisch) [inline README.md]: for problems regarding code and documentation.

File: README.md [hunk 5]
```diff
@@ -117,12 +117,26 @@ For specific toolchain installation, follow instructions in the
...
+with the code and documentation.
+
+### How to Ask
+
+Please include as much detail as you can that is relevant to your question, not
+only "this isn't working".
```
Comment (waehlisch) [inline README.md:135]: ... These details include

File: README.md (hunk 2 above)
Comment (waehlisch) [reply README.md:125]: yes, this comment was about rephrasing.

- [Forum](https://forum.riot-os.org) is the default place to ask for help. Our Forum provides an archive of prior questions and answers.

File: README.md (hunk 3 above)
Comment (waehlisch) [reply README.md:127]: thanks for this constructive comment.

easy to solve: For chats, we use [...]

**Comment by jkarinkl** [comment]
I wonder if the part you now call should be called 'Communications', with a sub part into 'asking for help'. Since you sum up the communication channels used (forum, matrix chat, GitHub). See this [README-template](https://github.com/cncf/project-template/blob/main/README-template.md) for an overview of communications. 

The used social media can also be summed up here (mastadon, youtube, ..?)

The sub part of communications will then be 'asking for help', stating that the forum is the main channel with the chat for small questions + how to ask.

**Comment by jkarinkl** [comment]
Can this PR also be labeled 'Community: good first issue'?

File: README.md [hunk 6]
```diff
@@ -117,12 +117,26 @@ For specific toolchain installation, follow instructions in the
   version of the documentation is uploaded daily to
   [doc.riot-os.org](https://doc.riot-os.org).
 
-## Forum
-Do you have a question, want to discuss a new feature, or just want to present
-your latest project using RIOT? Come over to our [forum] and post to your hearts
-content.
+## Asking for help
```
Comment (kfessel) [inline README.md:120]:
```suggestion
## Community

### how to get in contact
forum, matrix, webpage 

### Asking for help
You might already find your question answered in our forum or in an Gitub issue. You can search both of them.
```

**Review by Teufelchen1 — APPROVED** [review]

**Comment by bergzand** [comment]
Rebased!

**Pull request merged**